app/presenters/menu/manager.rb

```diff
@@ -73,8 +73,8 @@ class InvalidMenuDefinition < Exception
 
     def initialize
```

Comment:
Menu::Manager itself is a singleton and is the only consumer of Menu::YamlLoader and Menu::CustomLoader.  Since Menu::Manager  is a singleton is it effectively a cache in and of itself, so we don't need a _second_ cache in YamlLoader nor in CustomLoader.